### PR TITLE
[ch10000] metadata template object

### PR DESCRIPTION
**Pull request by jmorel**

### CODE REVIEW

File: source/includes/_dataset_metadata.md [hunk 1]
```diff
@@ -26,14 +30,44 @@ This object stores both the definition and the value of a given metadata.
...
 In the case of federated and harvested datasets, metadata values are automatically collected from the remote source, showing up in the object as `remote_value`. You can however override this value with your own, specifying it in `value` and setting the `override_remote_value` flag to `True`. This flag determines which value will show up in the explore API output.
 
+## The metadata template object
+
+> Example object
+
+```json
+{
+    "name": "default",
+    "title": "Default metadata",
```
Comment (remilejeune) [inline source/includes/_dataset_metadata.md]: @jmorel why not Label to be consistent ?

Comment (jmorel) [reply source/includes/_dataset_metadata.md]: 👍

Comment (jmorel) [reply source/includes/_dataset_metadata.md]: I was also thinking of changing "type" into "purpose"
What do you think?

Comment (remilejeune) [reply source/includes/_dataset_metadata.md]: I do not like type but purpose: basic is weird :(
namespace ?

Comment (jmorel) [reply source/includes/_dataset_metadata.md]: The weird part there is "basic" :) Namespace is too technical and not descriptive enough of the reason why this needs to exist. I say purpose and we rename basic into general/generic

Comment (jmorel) [reply source/includes/_dataset_metadata.md]: @Aquassaut any thought on the proper wording?

Comment (Aquassaut) [reply source/includes/_dataset_metadata.md]: Purpose and general seem like the logical thing

**Pull request merged**